presto-docs/src/main/sphinx/admin/kerberos.rst

```diff
@@ -0,0 +1,73 @@
+===========================
+CLI Kerberos Authentication
+===========================
+
+Presto provides limited experimental support for Kerberos authentication when command-line interface is used.
+
+Presto coordinator node configuration
+-------------------------------------
+
+To use Kerberos based authentication following configuration properties must defined for Presto coordinator node:
+
+.. code-block:: none
+
+    http.server.authentication.enabled=true
+    http.server.authentication.krb5.service-name=presto.coordinator.https
+    http.authentication.krb5.config=/etc/krb5.conf
+    http-server.https.enabled=true
+    http-server.https.port=7778
+    http-server.https.keystore.path=/etc/presto_keystore.jks
+    http-server.https.keystore.key=keystore_password
+
+* ``http.server.authentication.enabled``: Defines that Presto authentication should be enabled. Must be set to ``true``
+* ``http.server.authentication.krb5.service-name``: Defines Kerberos server name for Presto coordinator
+* ``http.authentication.krb5.config``: Kerberos configuration file
+* ``http-server.https.enabled``: Specifies if Presto server should listen for HTTPS
+* ``http-server.https.port``: HTTPS server port
+* ``http-server.https.keystore.path``: Path of keystore file to be used by HTTPS server
+* ``http-server.https.keystore.key``: Keystore password
+
+Presto CLI execution
+--------------------
+
+To use Kerberos on client side following extra parameters must be passed to presto-cli:
+
+.. code-block:: none
+
+    --enable-authentication
+    --krb5-config-path /etc/krb5.conf
+    --krb5-remote-service-name presto.coordinator.https
+    --keystore-path /etc/java_keystore.jks
+    --keystore-password the_keystore_password
+
+* ``--enable-authentication``: Defines that Presto authentication should be enabled.
+* ``--krb5-config-path``: Kerberos configuration file
+* ``--krb5-remote-service-name``: Presto coordinator Kerberos service name
+* ``--keystore-``: Path of keystore file to be used by HTTPS server
+* ``--keystore-password``: Keystore password
+
+Implementation access controller
+--------------------------------
+
+Additionally user need to provide implementation of access controller.
+This is done by implementing `SystemAccessControlFactory`` and ``SystemAccessControl`` interfaces.
+
+``SystemAccessControlFactory`` is responsible for creating ``SystemAccessControl`` instance and defining name later used
```

Comment:
a ``SystemAccessControl`` instance and defining a name

Also, it's a little unclear what kind of name it is defining